content/applications/finance/accounting/fiscal_localizations/localizations/italy.rst

```diff
@@ -0,0 +1,84 @@
+=====
+Italy
+=====
+
+**This documentation is still in the process of being written and might be subject to change.**
+
+Allow the user to generate the EDI document for Italian invoicing.
+
+This module allows the creation of the EDI documents and the communication with the SDICoop
+web-service of FatturaPA to send/receive invoices and notifications.
+
+.. warning::
+  Once this module is installed, it's no longer possible to send invoice via PEC
+  mail.
```

Comment:
In our RST guidelines, the indentation is aligned with the directive. So usually 3 white spaces, same for the image options.
https://www.odoo.com/documentation/14.0/contributing/documentation/rst_guidelines.html#be-consistent-with-indentation

Also, you break the line only once you reach the 100th character

+ invoices plural (as in my previous comment)

```suggestion
.. warning::
   Once this module is installed, it's no longer possible to send invoices via PEC mail.
```